core/src/main/java/hudson/model/User.java

```diff
@@ -688,7 +688,7 @@ private boolean relatedTo(@NonNull AbstractBuild<?, ?> b) {
     }
 
     public @Override String toString() {
-        return fullName;
+        return id;
```

Comment:
not going to have any impacts on the UI of any of the major auth plugins?

(assume not but just checking)

Reply:
I hope not - most of the tests passed (did not run ATH) - and any security authN/AuthZ plugin using `toString()` has a huge bug anyway as `fullNames` can change on a whim

I anticipate some test failures in some plugins if they are checking logs contain some message (as `"Jane Doe"` will now be `"janedoe"` or similar) but I am not expecting production code to break - but I also have not actively gone looking.

Reply:
NB: "most" above was because I was running on windows and there are known failures ;(

The ATH run as part of the PR passed according to Jenkins - so that is a very good sign that there is not a "huge ammount" of sillyness going on in the codebase

Reply:
There's very minimal ath tests run on PRs: https://github.com/jenkinsci/acceptance-test-harness/search?q=SmokeTest&unscoped_q=SmokeTest

Reply:
an internal run on CloudBees infrastructure showed no new failing ATHs and no regressions in the PCT of the plugins we support.

There where a large amount of failures - mainly due to a scrolling issue with the latest core and ATH (should be fixed in https://github.com/jenkinsci/acceptance-test-harness/pull/586) but I now consider this tested.